helm/service-mesh-bundle/values.yaml

```diff
@@ -90,3 +90,16 @@ apps:
     version: 0.11.0
     userConfig: {}
     extraConfigs: []
+  linkerd-certs:
+    appName: linkerd-certs
+    chartName: linkerd-certs
+    catalog: giantswarm-playground-test
+    enabled: true
```

Comment:
let's keep it disabled by default for now. you can enable on our tests with a values configMap

Reply:
done

Reply:
Can we get a real release of the `linked-certs` app in the `giantswarm-playground` catalog? :) At least I would not rely on the `-test` catalog for references, especially if we are releasing a new version of the bundle soon.